### PR TITLE
Transaction feed price service

### PR DESCRIPTION
**Pull request by dievazqu**
Calculating `TokenAmount.localAmount` using `PricesServices`

### CODE REVIEW

File: src/prices/PricesService.ts [hunk 1]
```diff
@@ -115,12 +127,16 @@ export default class PricesService {
     )
   }
 
-  private async cUSDToLocalCurrency(
+  private async USDToLocalCurrency(
     localCurrency: string,
     date: Date,
   ): Promise<BigNumber> {
-    return await this.exchangeAPI.getExchangeRate({
-      sourceCurrencyCode: this.cUSDAddress,
```
Comment (gnardini) [inline src/prices/PricesService.ts:123]: We used to pass an address but now a symbol? Looks like a weird change, does it not break anything?

**Review by gnardini — APPROVED** [review]
Looks good! Just a bit confused about the address vs symbol we pass as `sourceCurrencyCode`

File: src/prices/PricesService.ts [hunk 2]
```diff
@@ -115,12 +127,16 @@ export default class PricesService {
     )
   }
 
-  private async cUSDToLocalCurrency(
+  private async USDToLocalCurrency(
```
Comment (jeanregisser) [inline src/prices/PricesService.ts:130]: nit: functions should use camelCase
```suggestion
  private async usdToLocalCurrency(
```

**Review by jeanregisser — APPROVED** [review]
LGTM 👍

File: src/prices/PricesService.ts (hunk 1 above)
Comment (dievazqu) [reply src/prices/PricesService.ts:123]: I don't know why I did like that in the past 😅 
The latest version is the one that works :)

**Pull request merged**